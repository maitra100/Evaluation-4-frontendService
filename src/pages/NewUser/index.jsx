import React from 'react';
import './newuser.css';
import ImageContainer from '../../components/ImageContainer';
import RegisterUser from '../../components/RegisterUser';

function NewUser() {
  return (
    <div id="frontpage">
      <ImageContainer />
      <RegisterUser />
    </div>
  );
}

export default NewUser;
