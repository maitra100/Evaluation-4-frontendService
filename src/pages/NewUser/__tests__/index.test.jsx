import renderer from 'react-test-renderer';
import React from 'react';
import NewUser from '../index';

jest.mock('react-router-dom');

describe('Login Section', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<NewUser />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
