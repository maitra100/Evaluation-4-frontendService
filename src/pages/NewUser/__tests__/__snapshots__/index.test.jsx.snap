// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Section renders correctly 1`] = `
<div
  id="frontpage"
>
  <div
    id="loginLeft"
  >
    <div
      id="text1"
    >
      Design APIs fast,
    </div>
    <div
      id="text2"
    >
      Manage content easily
    </div>
    <img
      alt="frontpic"
      id="frontimage"
      src="frontpic.png"
    />
  </div>
  <div
    id="loginRight"
  >
    <div
      id="loginText"
    >
      <strong>
        Register to your CMS+ account
      </strong>
    </div>
    <div
      className="loginUser"
    >
      <label
        className="inputLabel"
        htmlFor="email"
      >
        Email
      </label>
      <input
        className="detailInput"
        onChange={[Function]}
        type="text"
        value=""
      />
      <label
        className="inputLabel"
        htmlFor="password"
      >
        Password
      </label>
      <input
        className="detailInput"
        onChange={[Function]}
        type="password"
        value=""
      />
    </div>
    <div
      id="submitButton"
    >
      <button
        id="submit"
        onClick={[Function]}
        type="submit"
      >
        Sign Up
      </button>
    </div>
  </div>
</div>
`;
